Translate 2020 Performance chapter for Traditional Chinese

CYBAI opened the pull request:
This is part of #1106

---

I'd like to also ask some friends to help to review the translation in case that this post is too long 🙏 

For better review experience, I committed the English version in the second commit first and then translate it in the third commit. So, you can review the 3rd commit for the translation. Wish it will help for the review experience!

cc @ArvinH @xu3u4 @jjjjackson 

---

@bazzadp Please let me know if there's any technical part I need to update or fix!

Btw, I found `Taiwan` is wrongly written as `Taiwan, Province of China` in the charts in the post. Could you help me to update them in those charts? Thank you.

FWIW, in 2019, `Taiwan` was written as `Taiwan` so I'm not sure why it's changed in 2020.

| 2019 | 2020 |
| --- | --- |
| <img width="587" alt="image" src="https://user-images.githubusercontent.com/6782666/102478436-ed976880-40a0-11eb-92f9-c8f55bdca195.png"> |  <img width="637" alt="image" src="https://user-images.githubusercontent.com/6782666/102478458-f4be7680-40a0-11eb-8c5c-09e8c0adafad.png"> |

Zuckjet commented:
Taiwan, Province of China is absolutely right, you shouldn't make this to become political problems.

ByronLian commented:
> Taiwan, Province of China is absolutely right, you shouldn't make this to become political problems.
I don't see any political word in @CYBAI 's comment. He only mentioned "Taiwan is wrongly written as Taiwan, Province of China", so what's your problem ?

Zuckjet commented:
why "Taiwan, Province of China" is wrong ?

andy0130tw commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+discuss: 2045
+results: https://docs.google.com/spreadsheets/d/164FVuCQ7gPhTWUXJl1av5_hBxjncNi0TK8RnNseNPJQ/
+queries: 09_Performance
+featured_quote: 糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障。今年的全球疫情那些已經存在的屏障。
```
The quote is inconsistent with the article; should be: 
> 但糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障。今年的全球性疫情更凸顯了那些已經存在的屏障。

CYBAI replied:
Thanks for catching!

因為在 quote 沒有用到 `but`，所以待會修改我會把第一個「但」拿掉！謝謝！

andy0130tw replied:
哇，我沒有注意到原文的引文就跟文章的句子不一樣，如果用本來的版本的話，中文的語句習慣會把倒數第二個句號改成冒號。

CYBAI replied:
原文跟引文主要應該就差在開頭的 but XD 我還是用了文章中的那段句子，然後把第二個句號改成全型分號了！（在這邊用分號好像比較順？）

bazzadp reviewed (changes requested):
Thanks for this - looks very good!

Not gone through it all but made some comments that should be considered for all the translation. Will review again after you've addressed these issues throughput the text.

bazzadp commented:
> > Taiwan, Province of China is absolutely right, you shouldn't make this to become political problems.
> 
> I don't see any political word in @CYBAI 's comment. He only mentioned "Taiwan is wrongly written as Taiwan, Province of China", so what's your problem ?

I raised this before in another chapter (more for readability that any political concern) and this is a relevant point of the discussion:

> Country names are kind of a sensitive topic. These names are sourced from Wikipedia and included in the underlying CrUX function that maps country codes to names, so I think keeping them would be the most consistent thing, even though I agree their formatting can be a bit strange.

So those are the "country names" we use in the Web Almanac even though you might not all agree with them.

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+reviewers: [borisschapira, rviscomi, obto, noamr, Zizzamia, exterkamp]
+analysts: [max-ostapenko, dooman87]
+translators: [cybai]
+thefoxis_bio: Karolina 是 <a href="https://calibreapp.com/">Calibre</a> 的 Product Design Lead。她主要負責打造最全面的速度監測平台。她同時也是為您蒐集效能相關訊息與資訊的 <a href="https://perf.email/">Performance Newsletter</a> 發起人之一。Karolina 也<a href="https://calibreapp.com/blog/category/web-platform">時常寫有關於效能如何影響使用者體驗的文章</a>。
```
I personally prefer to keep `Performance Newsletter` though, I would like to see how other translators think about this. If all agree with translating it, I'll update this one! Thanks!

Will update `Product Design Lead` later, thanks!

bazzadp replied:
OK but let's add `lang="en"` then:

```suggestion
thefoxis_bio: Karolina 是 <a href="https://calibreapp.com/">Calibre</a> 的 Product Design Lead。她主要負責打造最全面的速度監測平台。她同時也是為您蒐集效能相關訊息與資訊的 <a href="https://perf.email/" lang="en">Performance Newsletter</a> 發起人之一。Karolina 也<a href="https://calibreapp.com/blog/category/web-platform">時常寫有關於效能如何影響使用者體驗的文章</a>。
```

CYBAI replied:
Thanks for your reminder :)

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+results: https://docs.google.com/spreadsheets/d/164FVuCQ7gPhTWUXJl1av5_hBxjncNi0TK8RnNseNPJQ/
+queries: 09_Performance
+featured_quote: 糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障。今年的全球疫情那些已經存在的屏障。
+featured_stat_1: 25%
+featured_stat_label_1: Sites with good FCP on mobile
+featured_stat_2: 18%
+featured_stat_label_2: Sites with good TTFB on mobile
+featured_stat_3: 4%
+featured_stat_label_3: Sites with unchanged Performance Score in LH6
```
Yes! Thanks for catching it!

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+featured_stat_label_3: Sites with unchanged Performance Score in LH6
+---
+
+## Introduction
```
Yes! I will update some more titles which is not related to specific terms! Thanks!

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+
+效能調校工具在近幾年持續成長來描繪出各式各樣的使用者體驗，並使工程師更容易找到問題的根本原因。從[去年的效能章節](../2019/performance)開始，在這個領域中，已經有許多重要的開發貢獻改變了我們如何監控速度的方式。
+
+隨著很受歡迎的品質檢測工具 Lighthouse 釋出 Lighthouse 6 後，[眾所皆知的效能分數（Performance Score）背後的演算法已經有了重大的改動](https://calibreapp.com/blog/how-performance-score-works)，因此分數也隨之變動。新釋出的[Core Web Vitals](https://calibreapp.com/blog/core-web-vitals)提供一套新的衡量標準來描述不同面向的使用者體驗。未來他將成為搜尋排行的要素之一，也將促使開發社群聚焦於新的速度準則。
```
I see! Thanks for noting this!

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+
+讓我們一起一探究竟吧。
+
+## Lighthouse Performance Score
```
Thanks for the example about how to add an extra attribute for anchor link! Will update!

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,389 @@
+{{ figure_markup(
+  image="performance-change-in-lighthouse-score.png",
+  caption="Lighthouse 版本 5 跟版本 6 之間的效能分數（Performance Score）差異",
+  description="Column chart showcasing the change in Performance Score between versions 5 and 6. The highest percentage of websites (4%) observed no changes to the score, and the number of sites with score decreases outweighs score improvements.",
```
Thanks!

bazzadp commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+part_number: II
+chapter_number: 9
+title: 效能
+description: 2020 Web Almanac 的效能章節涵蓋了 <span lang="en">Core WebVitals</span>、<span lang="en">Lighthouse</span> <span lang="en">Performance Score</span>、<span lang="en">First Contentful Paint</span> (FCP) 以及 <span lang="en">Time to First Byte</span> (TTFB)。
```
The description is plain text not HTML so can we remove spans from here and revert this back? Sorry not obvious what can take HTML and what can't!

```suggestion
description: 2020 Web Almanac 的效能章節涵蓋了 Core WebVitals、Lighthouse Performance Score、First Contentful Paint (FCP) 以及 Time to First Byte (TTFB)。
```

CYBAI replied:
Thanks for catching it! I was not sure if they should be wrapped with `lang=en` so just tried this way first. Let me revert them! Thank you!

CYBAI commented:
@bazzadp Addressed your comments in the fixup commit! Please let me know if I miss anything! Thanks!

> So those are the "country names" we use in the Web Almanac even though you might not all agree with them.

I see. Thanks for your explanation.

bazzadp commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+queries: 09_Performance
+featured_quote: 糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障；今年的全球性疫情更凸顯了那些已經存在的屏障。
+featured_stat_1: 25%
+featured_stat_label_1: 擁有 <span lang="en">Good</span> 的 FCP 的手機版網頁
+featured_stat_2: 18%
+featured_stat_label_2: 擁有 <span lang="en">Good</span> 的 TTFB 的手機版網頁
+featured_stat_3: 4%
+featured_stat_label_3: 在 <span lang="en">Lighthouse</span> 6 時效能分數（<span lang="en">Performance Score</span>）未改變的網站
```
We really should support `<span lang="en"></span>` in our featured stats, but currently can't :-(
I must look into why not again but in the meantime need to remove these for now unfortunately.

```suggestion
featured_stat_label_1: 擁有 Good 的 FCP 的手機版網頁
featured_stat_2: 18%
featured_stat_label_2: 擁有 Good 的 TTFB 的手機版網頁
featured_stat_3: 4%
featured_stat_label_3: 在 Lighthouse 6 時效能分數（Performance Score）未改變的網站
```

CYBAI replied:
Thanks for catching it!

CYBAI commented:
@bazzadp Addressed these 2 comments in the second fixup commit! If they are okay, I will squash them into the 3rd commit for better translation review experience :) Thanks!

bazzadp commented:
> @bazzadp Addressed these 2 comments in the second fixup commit! If they are okay, I will squash them into the 3rd commit for better translation review experience :) Thanks!

We squash the commits on merge to `main` so don't worry about that.

And btw you can bulk accept suggestions in the File tab which is usually easier than accepting them one by one. Though where I have only given an example you're better to do them offline and submit as one commit.

CYBAI commented:
> > @bazzadp Addressed these 2 comments in the second fixup commit! If they are okay, I will squash them into the 3rd commit for better translation review experience :) Thanks!
> 
> We squash the commits on merge to `main` so don't worry about that.
> 
> And btw you can bulk accept suggestions in the File tab which is usually easier than accepting them one by one. Though where I have only given an example you're better to do them offline and submit as one commit.

Ah, I see! Thanks for the information!

CYBAI commented:
Ah, I mean squashing the fixup commits for review purpose that the second commit is the English version so that translation reviewers can just use Split view in Files while reviewing which might be easier review. so just squashed those fixup commits so that translation reviewers can check the last commit directly 🙏 still thanks for the tip!

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+## 簡介
 
-There is an unquestionable, detrimental effect that slow speed has on overall user experience, and consequently, conversions. But poor performance doesn't just cause frustration or negatively affects business goals—it creates real-life barriers to entry. This year's global pandemic [made those existing barriers even more apparent](https://www.weforum.org/agenda/2020/04/coronavirus-covid-19-pandem ...
+緩慢的速度無庸置疑地對使用者體驗造成了負面的影響，因此對於轉換率也是。但糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障。今年的全球性疫情[更凸顯了那些已經存在的屏障](https://www.weforum.org/agenda/2020/04/coronavirus-covid-19-pandemic-digital-divide-internet-data-broadband-mobbile/)。隨著遠端學習、遠端工作還有遠端社交興起，人們的生活重心突然就被移到網路世界。微弱的訊號以及缺乏資源取得可用設備將這個轉變顯得更痛苦，即使不是非常嚴重，也已經對很多人造成影響。針對訊號強度、設備以及網路速度的不平等性儼然成為全球性現實生活的測試。
```
> 緩慢的速度無庸置疑地對使用者體驗造成了負面的影響，因此對於轉換率也是

後半段翻成：「也影響了轉換率」或者是「進而影響了轉換率」，會不會比較通順？

aszx87410 replied:
> 他還築起了現實生活中人們與網路世界的屏障

不確定有沒有一定要用「它」還是用「他」也可以

CYBAI replied:
> 後半段翻成：「也影響了轉換率」或者是「進而影響了轉換率」，會不會比較通順？

「進而影響了轉換率」感覺比較通順！感謝！

> 不確定有沒有一定要用「它」還是用「他」也可以

同意用「它」比較適合 ++ 👍

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+緩慢的速度無庸置疑地對使用者體驗造成了負面的影響，因此對於轉換率也是。但糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障。今年的全球性疫情[更凸顯了那些已經存在的屏障](https://www.weforum.org/agenda/2020/04/coronavirus-covid-19-pandemic-digital-divide-internet-data-broadband-mobbile/)。隨著遠端學習、遠端工作還有遠端社交興起，人們的生活重心突然就被移到網路世界。微弱的訊號以及缺乏資源取得可用設備將這個轉變顯得更痛苦，即使不是非常嚴重，也已經對很多人造成影響。針對訊號強度、設備以及網路速度的不平等性儼然成為全球性現實生活的測試。
 
-Performance tooling continues to evolve to portray those diverse aspects of user experience and make it easier to find underlying issues. Since [last year's Performance chapter](../2019/performance), there have been numerous significant developments in the space that have already transformed how we approach speed monitoring.
+效能調校工具在近幾年持續成長來描繪出各式各樣的使用者體驗，並使工程師更容易找到問題的根本原因。從[去年的效能章節](../2019/performance)開始，在這個領域中，已經有許多重要的開發貢獻改變了我們如何監控速度的方式。
```
> 改變了我們如何監控速度的方式

好像「改變了我們監控速度的方式」就可以了？加個「如何」感覺很英文XD

aszx87410 replied:
> 效能調校工具在近幾年持續成長來描繪出各式各樣的使用者體驗

這句感覺原文的 portray 不一定要直接翻，或許也可以翻成：「效能調校工具在近幾年持續演進，幫助工程師從不同面向去檢視使用者體驗，也使他們更容易找到問題的成因」

CYBAI replied:
> 這句感覺原文的 portray 不一定要直接翻，或許也可以翻成：「效能調校工具在近幾年持續演進，幫助工程師從不同面向去檢視使用者體驗，也使他們更容易找到問題的成因」

感謝！整篇文章用到蠻多 portray 一直有點不知道怎麼翻 😂 不過你翻出來這句蠻不錯的，但最後一句我在想如果是「也使問題的成因更容易被找到」會不會比較順？🤔

aszx87410 replied:
我自己是覺得好像差不多，都算順XD

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+效能調校工具在近幾年持續成長來描繪出各式各樣的使用者體驗，並使工程師更容易找到問題的根本原因。從[去年的效能章節](../2019/performance)開始，在這個領域中，已經有許多重要的開發貢獻改變了我們如何監控速度的方式。
 
-With a significant release of the popular quality auditing tool, Lighthouse 6, [the algorithm behind the famous Performance Score has changed significantly](https://calibreapp.com/blog/how-performance-score-works), and so did the scores. [Core Web Vitals](https://calibreapp.com/blog/core-web-vitals), a set of new metrics portraying different aspects of user experience, has been released. It will  ...
+隨著很受歡迎的品質檢測工具 <span lang="en">Lighthouse</span> 釋出 <span lang="en">Lighthouse</span> 6 後，[眾所皆知的效能分數（<span lang="en">Performance Score</span>）背後的演算法已經有了重大的改動](https://calibreapp.com/blog/how-performance-score-works)，因此分數也隨之變動。新釋出的[<span lang="en">Core WebVitals</span>](https://calibreapp.com/blog/core-web-vitals)提供一套新的衡量標準來描述不同面向的使用者體驗。未來他將成為搜尋排行的要素之一，也將促使開發社群聚焦於新的速度準則。
```
> 未來他將成為搜尋排行的要素之一

「要素之一」看起來也可以但總覺得有點小奇怪，如果是「未來它將會影響搜尋引擎的排行」呢？

CYBAI replied:
確實以中文翻法來說不用特別強調「要素之一」！

```diff
- 未來他將成為搜尋排行的要素之一，也將促使開發社群聚焦於新的速度準則。
+ 未來它將會影響搜尋引擎的排行，進而促使相關開發者社群聚焦於新的速度準則。
```

配合你翻的句子，感覺後面改成這樣會不會比較順呢？
（btw, 突然發現 the development community 只翻「開發社群」有點怪 XD 所以先暫定把它改成「相關開發者社群」）

aszx87410 replied:
有，改了之後看起來有比較好！

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+隨著很受歡迎的品質檢測工具 <span lang="en">Lighthouse</span> 釋出 <span lang="en">Lighthouse</span> 6 後，[眾所皆知的效能分數（<span lang="en">Performance Score</span>）背後的演算法已經有了重大的改動](https://calibreapp.com/blog/how-performance-score-works)，因此分數也隨之變動。新釋出的[<span lang="en">Core WebVitals</span>](https://calibreapp.com/blog/core-web-vitals)提供一套新的衡量標準來描述不同面向的使用者體驗。未來他將成為搜尋排行的要素之一，也將促使開發社群聚焦於新的速度準則。
 
-In this chapter, we will be looking at real-world performance data provided by the [Chrome User Experience Report (CrUX)](https://developers.google.com/web/tools/chrome-user-experience-report) through the lens of those new developments as well as analyzing a handful of other relevant metrics. It is important to note that due to the limitations of iOS, CrUX mobile results don't include devices run ...
+在此章節，我們將透過這些新工具來透視[<span lang="en">Chrome User Experience Report</span> (CrUX)](https://developers.google.com/web/tools/chrome-user-experience-report)提供的真實效能資料並分析相關的準則。在這邊必須特別註記，由於 iOS 的限制，CrUX 手機版的結果並不包含 Apple 手機作業系統的裝置。這個事實無可否認地影響了我們的分析，特別是當我們以國家為單位在檢視效能衡量標準時。
```
> 這個事實無可否認地影響了我們的分析，特別是當我們以國家為單位在檢視效能衡量標準時

這個限制確實影響了我們的分析，尤其是在檢視各個國家的效能衡量標準的時候。

總覺得不管怎樣翻看起來都有點小奇怪XD 但「事實」應該可以不用照翻，我自己覺得沿用前面的「限制」應該會比較通順一點，後半段我就不知道怎麼翻比較好了

CYBAI replied:
看了你的句子之後我覺得我翻的超怪 😂 不過我想後面那句也許可以再修成

> 這個限制確實影響了我們的分析，尤其是在檢視各個國家的效能衡量標準的時候更能顯現出分析結果的差異。

這樣呢？

aszx87410 replied:
喔喔這樣感覺可以

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+在 2020 年五月時，[<span lang="en">Lighthouse</span> 6 被釋出](https://github.com/GoogleChrome/lighthouse/releases/tag/v6.0.0)。在此主版號更新中，受歡迎的效能檢測組針對效能分數（<span lang="en">Performance Score</span>）演算法做了顯著的改變。效能分數（<span lang="en">Performance Score</span>）可以說是檢測網站速度的最佳寫照 (<span lang="en">high-level portrayal</span>)。在 <span lang="en">Lighthouse</span> 6 之中，以六項（不是五項）衡量標準來評估分數：<span lang="en">First Meaningful Pai ...
 
-The new scoring algorithm is prioritizing the new generation of performance metrics: Core Web Vitals and deprioritizing First Contentful Paint (FCP), Time to Interactive (TTI) and Speed Index, as their score weight decreases. The algorithm also now distinctly emphasizes three aspects of user experience: *interactivity* (Total Blocking Time and Time to Interactive), *visual stability* (Cumulative  ...
+新的分數演算法調整了新一代的效能衡量標準的優先度：<span lang="en">Core WebVitals</span>，以及降低 <span lang="en">First Contentful Paint</span> (FCP)、<span lang="en">Time to Interactive</span> (TTI) 以及 Speed Index 的優先度並減少他們的分數權重。此演算法現在也會將三個層面的使用者體驗個別強調： **互動性**（<span lang="en">Total Blocking Time</span> 及 <span lang="en">Time to Interactive</span>）、**視覺穩定性**（<span lang="en">Cumulative Layout Shift</span>）以及**體感載入速度**（<span la ...
```
> 此演算法現在也會將三個層面的使用者體驗個別強調

此演算法現在也會個別強調三個層面的使用者體驗

CYBAI replied:
感謝！

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+新的分數演算法調整了新一代的效能衡量標準的優先度：<span lang="en">Core WebVitals</span>，以及降低 <span lang="en">First Contentful Paint</span> (FCP)、<span lang="en">Time to Interactive</span> (TTI) 以及 Speed Index 的優先度並減少他們的分數權重。此演算法現在也會將三個層面的使用者體驗個別強調： **互動性**（<span lang="en">Total Blocking Time</span> 及 <span lang="en">Time to Interactive</span>）、**視覺穩定性**（<span lang="en">Cumulative Layout Shift</span>）以及**體感載入速度**（<span la ...
 
-Additionally, the score is now calculated using different reference points for desktop and mobile. What this means in practice is that it will be less forgiving on desktop (expecting fast websites) and more relaxed on mobile (since benchmark performance on mobile is less quick than desktop). You can compare your Lighthouse 5 and 6 score difference in [the Lighthouse Score calculator](https://goog ...
+此外，該分數會根據桌機和手機的不同來使用不一樣的參考標準做計算。 實際上來說，這代表對桌機的標準較嚴格（預期較快的網頁）而在手機上標準較寬鬆（因為手機效能比桌機較慢）。你可以透過 [<span lang="en">Lighthouse</span> 分數計算機](https://googlechrome.github.io/lighthouse/scorecalc/)來比較 <span lang="en">Lighthouse</span> 5 及 6 之間的差異。所以，分數上到底如何改變了呢？
```
> 所以，分數上到底如何改變了呢？

1. 所以，分數上到底改變了什麼呢？
2, 所以，分數上到底有什麼變動呢？

ArvinH replied:
🤔  「如何改變」似乎比較貼近原意

還是 「所以，在評分上到底做了什麼變動呢？」？

CYBAI replied:
感謝兩位！

綜合兩位的翻譯，讓我想到也許這樣翻不錯 👀?

> 所以，在分數上發生了怎麼樣的變動呢？

（這個我會先採用 Arvin 的翻法，再在 fixup commit 一起討論 🙇）

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -98,52 +98,52 @@ In the chart above, we can observe that between 43% and 53% of websites have goo
 }}
 
-Since LCP is a timing showcasing when the largest above-the-fold element has been rendered (including imagery, videos or block-level elements containing text), it is not surprising that the slower the network, the more significant portion of measurements are poor.
+LCP 能夠顯現出計時方面的效能優點，當最大的 <span lang="en">above-the-fold element</span> 被 <span lang="en">render</span> 時（包括圖像、影片或 block 層級包含文字的元素），一點也不意外越慢的網路造成網路連線不佳的比例越高。
```
因為 LCP 是用來計算最大的 above-the-fold element 被 render 時（包括圖像、影片或 block 層級包含文字的元素）的時間，因此毫不意外地，網路速度越慢，LCP 的表現也越糟。

這句的意思感覺會比較像是這樣？

CYBAI replied:
確實原本的翻法有點奇怪... 感謝！

CYBAI replied:
```diff
- 因為 LCP 是用來計算最大的 above-the-fold element 被 render 時（包括圖像、影片或 block 層級包含文字的元素）的時間，因此毫不意外地，網路速度越慢，LCP 的表現也越糟。
+ 因為 LCP 是用來計算最大的 above-the-fold element 被 render 時（包括圖像、影片或 block 層級包含文字的元素）的所需時間，因此無庸置疑地，網路速度越慢，LCP 的表現也越糟。
```

小修了一下改成這樣 ☝️

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -98,52 +98,52 @@ In the chart above, we can observe that between 43% and 53% of websites have goo
+## <span lang="en">Core WebVitals</span>: <span lang="en">Cumulative Layout Shift</span> {core-web-vitals-cumulative-layout-shift}
 
-Cumulative Layout Shift (CLS) quantifies how much elements within the viewport move around during the page visit. It helps pinpoint the degree to which unexpected movement occurs on your websites to grade the user experience, rather than attempting to measure a specific part of interaction with the help of a unit like seconds or milliseconds.
+<span lang="en">Cumulative Layout Shift</span> (CLS) 量化了在訪問網頁時，可視區域（viewport）中有多少元素移動了。並非透過時間性單位（秒或毫秒）來計算特定區快的互動性，CLS 協助我們找出訪問網頁時不如預期的頁面移動並以此來評分使用者體驗。
```
Cumulative Layout Shift (CLS) 跟其他用秒或毫秒來衡量網頁可互動性的指標不同。它量化了在訪問網頁時，可視區域（viewport）中有多少元素移動了，藉此協助我們找出訪問網頁時未預期的元素移動並以此來衡量使用者體驗的好壞。

CYBAI replied:
哇！我原本太按照英文的語序了.... 這樣翻順多了！感謝！！！

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -152,52 +152,52 @@ According to CrUX data, both in cases of desktop and mobile devices, more than h
+相較其他評比，手機及桌機均以高百分比分佈在 <span lang="en">Good</span> 比較不常見。在桌機中，基於 75% 網頁分佈來說，有 100% 的網站得到相當快的 FID 時間差，這也意味著體驗到互動延遲的使用者極少。
 
-On mobile, 80% of sites are graded as good. A likely explanation is the reduced CPU capabilities in comparison to desktop, network latency on mobile (causing a delay in script download and execution) as well as battery efficiency and temperature limitations, capping the CPU potential of mobile devices. All of which directly affect interactivity metrics like FID.
+在手機中，有 80% 的網站被評比為 <span lang="en">Good</span>。較合理的解釋可能是，相較於桌機，手機的 CPU 性能較低，而網路延遲（導致 <span lang="en">script</span> 下載及執行的延遲）、電池功率以及溫度限制都是消耗 CPU 的潛在因素。以上所述因素皆有可能影響 FID 類型的互動性衡量標準。
```
> 以上所述因素皆有可能影響 FID 類型的互動性衡量標準。

以上所述因素皆有可能影響像是 FID 這種類型的互動性衡量標準。

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -206,34 +206,34 @@ On mobile, 80% of sites are graded as good. A likely explanation is the reduced
 }}
 
-We can observe a direct correlation between network speed and fast FID, ranging from 73% on 2G to 87% on 4G networks. Faster networks will aid in speedier script download, which consequently speeds up the beginning of the parsing and fewer tasks blocking the main thread. It is optimistic to see such results, especially when the ratio of poorly rated site experiences doesn't exceed 5%.
+從 2G 的 73% 到 4G 的 87%，我們可以觀察到網路速度與快速的 FID 有直接關係。較快的網路可以幫助較迅速的 <span lang="en">script</span> 下載，固然也就加快了 <span lang="en">parsing</span> 的開始以及使得較少的 <span lang="en">task block</span> 在 <span lang="en">main thread</span> 上。這是一份很樂觀的結果報告，特別是評比不佳的網站比例不超過 5%。
```
> 我們可以觀察到網路速度與快速的 FID 有直接關係

我們可以觀察到網路速度與 FID 有直接關係

好像不太需要「快速」兩個字，因為後面段落會說明

aszx87410 replied:
> 較快的網路可以幫助較迅速的 <span lang="en">script</span> 下載，固然也就加快了 <span lang="en">parsing</span> 的開始以及使得較少的 <span lang="en">task block</span> 在 <span lang="en">main thread</span> 上。

較快的網路可以較快地下載 script，也因此提早了開始 parsing 的時間以及減少了阻塞 main thread 的 task

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -206,34 +206,34 @@ On mobile, 80% of sites are graded as good. A likely explanation is the reduced
+## <span lang="en">First Contentful Paint</span> {first-contentful-paint}
 
-First Contentful Paint (FCP) measures the first time at which the browser rendered any text, image, non-white canvas or SVG content. FCP is a good indicator of perceived speed as it portrays how long people wait to see the first signs of a site loading.
+<span lang="en">First Contentful Paint</span> (FCP) 用於測量瀏覽器第一次 <span lang="en">render</span> 任何文字、圖片、非白色 <span lang="en">canvas</span> 或是 SVG 內容。FCP 能夠描繪出「在網站載入時，使用者需要等待多久時間才能看到第一個內容被呈現」，因此這是一項測量使用者體感速度的好指標。
```
我一直覺得「好指標」聽起來有點小奇怪，但也不確定怎樣翻會比較好

有一種可能是直接把好拿掉：

1. 因此這是一項測量使用者體感速度的指標

另一種是把 good 翻成「適合」

1. 因此這是一項適合測量使用者體感速度的指標

CYBAI replied:
確實這邊的 good  是「適合」的意思！讓我用「適合」的版本來改！感謝！

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -242,36 +242,36 @@ First Contentful Paint (FCP) measures the first time at which the browser render
 }}
 
-In the charts above, the FCP distributions are broken down by desktop and mobile. [Comparing to last year](../2019/performance#fcp-by-device), there are noticeably less average FCP readings, while the percentage of fast and slow user experiences has risen no matter the device type. We can still observe the same trend, where mobile users will experience slower FCP more frequently than desktop user ...
+在上表中，FCP 分佈被拆成桌機及手機。[相較去年結果](../2019/performance#fcp-by-device)，可以發現到較少 FCP 指數呈現為 <span lang="en">Average</span>，反而可以觀察到無論是哪種裝置，<span lang="en">Fast</span> 及 <span lang="en">Slow</span> 的比率都升高了。此外，也可以發現手機使用者相較桌機使用者更頻繁的體驗較慢的 FCP 的情況仍然存在。整體來說，使用者們較可能體驗好的或差的體驗，而不是普通的體驗。
```
> 整體來說，使用者們較可能體驗好的或差的體驗，而不是普通的體驗。

整體來說，使用者們的體驗較為兩極，不是好就是差。

或是：

整體來說，使用者們的體驗較為兩極，不是好就是差，很少有一般的體驗。

ArvinH replied:
+1 for 整體來說，使用者們的體驗較為兩極，不是好就是差，很少有一般的體驗。

CYBAI replied:
我也覺得後者不錯！感謝兩位！

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -280,58 +280,58 @@ Numerous factors can delay paints, such as server latency (measured by a handful
+在我們開始分析前，值得一提的是，2019 年效能章節的 <span lang="en">Good</span> 及 <span lang="en">Poor</span> 評估標準和 2020 年不同。在 2019 年的標準中，FCP 低於 1 秒的網站被分類為 <span lang="en">Good</span>，高於 3 秒的才會被分類為 <span lang="en">Poor</span>。而在 2020 年的標準中，評分標準變成低於 1.5 秒為 <span lang="en">Good</span>、高於 2.5 秒為 <span lang="en">Poor</span>。
 
-This change means that the distribution would shift towards more "good" and "poor" rated websites. We can observe that trend compared to [last year's results](../2019/performance#fcp-by-geography), as the percentage of good and poor websites rise. The top ten geographies with the highest rate of fast websites remain relatively unchanged from 2019, with the addition of Czechia and Belgium and the  ...
+這個改變也代表著此數值分佈圖中被評為 <span lang="en">Good</span> 及 <span lang="en">Poor</span> 的網站都會變得更多。相較於[去年的結果](../2019/performance#fcp-by-geography)，我們可以觀察到此現象。以地理位置分佈來說，前十名最快的網站的國家與 2019 相同，但捷克與比利時的排名變前面且美國與英國的名次下降了。韓國以幾近於去年兩倍的 62% 的超快 FCP 數值領先在第一名（可以猜測受惠於評估標準的改變）。前幾名的國家也幾乎都是去年的 <span lang="en">Good</span> 的兩倍。
```
> 這個改變也代表著此數值分佈圖中被評為 <span lang="en">Good</span> 及 <span lang="en">Poor</span> 的網站都會變得更多。相較於去年的結果，我們可以觀察到此現象

這個改變也代表著此數值分佈圖中被評為 <span lang="en">Good</span> 及 <span lang="en">Poor</span> 的網站都會變得更多。我們可以從今年跟去年的結果比較中觀察到這個狀況

或可能也可以直接換順序：

相較於去年的結果，評估標準的變動帶來的影響是有更多的網站都被評為 <span lang="en">Good</span> 及 <span lang="en">Poor</span>。

CYBAI replied:
啊 直接換順序看起來比較順！感謝！

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -280,58 +280,58 @@ Numerous factors can delay paints, such as server latency (measured by a handful
+這個趨勢更進一步描繪出了數位差距，網路越慢且可能性能越差的裝置條件下的效能體驗越差。 改善慢速連線下的 FCP 與增強 TTFB 有直接關係，我們可以在 [Aggregate TTFB 根據連線方式來分析的 performance chart](#ttfb-by-connection-type) 觀察到 <span lang="en">Poor</span> TTFB = <span lang="en">Poor</span> FCP。
 
-The choice of [hosting provider](https://ismyhostfastyet.com/) or [CDN](https://www.cdnperf.com/) will have a cascading effect on speed. Making these decisions based on the fastest possible delivery will help in improving FCP and TTFB, especially on slower networks. FCP is also significantly affected by font load time, so [ensuring text is visible while web fonts are downloaded](https://web.dev/f ...
+對 [hosting provider](https://ismyhostfastyet.com/) 或是 [CDN](https://www.cdnperf.com/) 的選擇對於速度也許會有加乘效果。基於速度最快的可能選擇服務會幫助改善 FCP 及 TTFB，尤其是在網速較慢的情況。字體載入時間也深深影響了 FCP，所以[確保 Web Fonts 下載後是看得到的](https://web.dev/font-display/)也是值得一試的策略（特別是在較慢的網路通訊之下，下載這些資源都相當耗時）。
```
> 基於速度最快的可能選擇服務會幫助改善 FCP 及 TTFB，尤其是在網速較慢的情況。

選擇速度可能最快的服務會幫助改善 FCP 及 TTFB，尤其是在網速較慢的情況。

或是：

選擇速度可能最快的服務會幫助改善 FCP 及 TTFB，在網路較慢的情況下更是如此。

CYBAI replied:
後者順多了！感謝 👍

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -340,48 +340,48 @@ On desktop, 76% of websites have a "not good" TTFB, while on mobile, that percen
+## 總結
 
-User experience is not only a spectrum but also depends on a wide variety of factors. To attempt understanding the state of performance without excluding the sub-par, underprivileged experiences, we must approach it intersectionally. Each website visit tells a story. Our personal and country-level socioeconomic status dictates the type of device and internet provider we can afford. The geopositio ...
+使用者體驗不單單只是個光譜，他更是一個基於重重因素所產生的光譜。為了不能忽視低於效能衡量標準或是無法體驗適當效能的人們，我們一定要設法了解並分析各個面向的數據。每個網頁訪問都述說著各自的故事。我們個人的以及國家層級的經濟狀況也敘述著我們能夠負擔的裝置型態以及網路提供者（<span lang="en">internet provider</span>）。我們生活的地方也深深影響著我們的網路延遲時間（澳洲人時常感受到這個痛苦），而且該國家的經濟也反應了當地行動網路覆蓋率。我們會訪問怎麼樣的網頁呢？我們為什麼想訪問那些網頁呢？了解 context 非常的重要，這不僅僅是為了要分析資料，也是為了能夠開發有足夠同理心、在乎是否容易被訪問且提供了快速的使用者體驗的網頁給所有的使用者。
```
> 使用者體驗不單單只是個光譜，他更是一個基於重重因素所產生的光譜。

使用者體驗是一個基於重重因素所產生的光譜。

我自己覺得融合成一句好像也 ok

CYBAI replied:
> 重重因素是不是指種種因素的意思.

其實我本來是想要表達「多重因素」的意思 XD 不過以用法來說似乎「種種因素」比較常見！感謝！

> > 使用者體驗不單單只是個光譜，他更是一個基於重重因素所產生的光譜。
> 
> 使用者體驗是一個基於重重因素所產生的光譜。
> 
> 我自己覺得融合成一句好像也 ok

的確以中文來說照我的翻法會有種饒口的感覺，融合成一句聽起來蠻合理的，不過我在想作者應該有想要「強調」「種種因素」的意思，所以如果把「種種因素」變粗體呢？

> 使用者體驗是一個基於**種種因素**所產生的光譜。

（我會先用粗體版本修正！）

aszx87410 commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -340,48 +340,48 @@ On desktop, 76% of websites have a "not good" TTFB, while on mobile, that percen
+根據這些數據所知，可能因為我們養尊處優（中至高收入的國家、高薪水且持有新又性能好的裝置）無法時常體會各式各樣的情境（例如較慢的網路連接速度），我們更應該要花心力來為這各式各樣的情境去改善效能。這同時也替我們強調了，我們在加速初始瀏覽器繪製（LCP 及 FCP）和資源傳送（TTFB）都還有許多未完成的事情。這些效能問題時常聽起來像前端的問題，但其實許多重要的改善是可以透過適當的 <span lang="en">infrastructure</span> 選擇或是在後端做出適當的修改所達成。再次強調，使用者體驗是由各式各樣因素所組成的光譜，所以我們必須一視同仁。
 
-New metrics bring new lenses to analyze user experience through, but we must not forget existing signals. Let's focus on moving the needle in the areas that need the most improvement and will result in positive shifts in experience for most underserved. Fast and accessible internet is a human right.
+新的衡量標準為我們帶來了不一樣的視野來分析使用者體驗，但我們也莫忘既有的問題與數據。讓我們繼續在改善需要被加強的領域，還有為了將那些在低標準體驗的使用者們邁向好的使用者體驗繼續努力。快速且容易存取的網路是基本人權。
```
> 讓我們繼續在改善需要被加強的領域，還有為了將那些在低標準體驗的使用者們邁向好的使用者體驗繼續努力。

為了改善那些不良的使用者體驗，讓我們繼續努力，持續改進需要被加強的領域。

或是：

為了那些體驗不良的使用者們，讓我們繼續努力，持續改進需要被加強的領域。

aszx87410 replied:
話說「accessible」在整篇文章裡面目前是被翻成「可訪問」，如果翻成「無障礙」會不會比較好？

ArvinH replied:
> 話說「accessible」在整篇文章裡面目前是被翻成「可訪問」，如果翻成「無障礙」會不會比較好？

可以參考 MDN 的版本，無障礙似乎是比較普遍的譯法
https://developer.mozilla.org/zh-TW/docs/Web/Accessibility

CYBAI replied:
> 為了那些體驗不良的使用者們，讓我們繼續努力，持續改進需要被加強的領域。

感覺這樣比較符合原句！感謝！

> > 話說「accessible」在整篇文章裡面目前是被翻成「可訪問」，如果翻成「無障礙」會不會比較好？
> 可以參考 MDN 的版本，無障礙似乎是比較普遍的譯法
> https://developer.mozilla.org/zh-TW/docs/Web/Accessibility

感謝啊啊啊！！！我一直很困擾不知道這字該怎麼翻.... 不過的確「無障礙」很到位！（不知道原來 MDN 的 a11y 有繁中翻譯 👀）

bazzadp commented:
Can we add profiles to `config/2020.json` for  @ArvinH , @aszx87410 and @andy0130tw so they get credit for their help, in our Contributors page?

@CYBAI you get the credit on the chapter itself as think you did the majority of the work here - though feel free to change that, if that's not the case.

I've also created a @HTTPArchive/traditional-chinese-translators group and invited you all into that so you can easily ask the full team to review any future PRs rather than adding them one by one.

CYBAI commented:
Thanks to all the revewers! I haven't gone through all the comments above but I will try to fix them or discuss with you all soon 🙇 

> Can we add profiles to `config/2020.json` for @ArvinH , @aszx87410 and @andy0130tw so they get credit for their help, in our Contributors page?
> 
> @CYBAI you get the credit on the chapter itself as think you did the majority of the work here - though feel free to change that, if that's not the case.
> 

Yes! That sounds great to me! I will help to add them into the configuration! and also the translator of this chapter of course ;)

> I've also created a @HTTPArchive/traditional-chinese-translators group and invited you all into that so you can easily ask the full team to review any future PRs rather than adding them one by one.

Thank you so much :)

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -1,95 +1,95 @@
+## 簡介
 
-There is an unquestionable, detrimental effect that slow speed has on overall user experience, and consequently, conversions. But poor performance doesn't just cause frustration or negatively affects business goals—it creates real-life barriers to entry. This year's global pandemic [made those existing barriers even more apparent](https://www.weforum.org/agenda/2020/04/coronavirus-covid-19-pandem ...
+緩慢的速度無庸置疑地對使用者體驗造成了負面的影響，因此對於轉換率也是。但糟糕的效能並不只是讓使用者挫折或是對業務目標造成負面影響，他還築起了現實生活中人們與網路世界的屏障。今年的全球性疫情[更凸顯了那些已經存在的屏障](https://www.weforum.org/agenda/2020/04/coronavirus-covid-19-pandemic-digital-divide-internet-data-broadband-mobbile/)。隨著遠端學習、遠端工作還有遠端社交興起，人們的生活重心突然就被移到網路世界。微弱的訊號以及缺乏資源取得可用設備將這個轉變顯得更痛苦，即使不是非常嚴重，也已經對很多人造成影響。針對訊號強度、設備以及網路速度的不平等性儼然成為全球性現實生活的測試。
```
綜合我們三個的版本，我又想出了一個新的版本 😆

> 這儼然成為現實生活中的考驗，且更凸顯了全球在訊號強度、設備以及網路速度上的不平等

大家覺得呢？

（考慮到語句順不順的問題，我暫時會先採用 Arvin 的版本，如果覺得上面這個新版本不錯的話我再修正！感謝！）

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+
+## <span lang="en">Core WebVitals</span>: <span lang="en">Largest Contentful Paint</span> {core-web-vitals-largest-contentful-paint}
+
+<span lang="en">Largest Contentful Paint</span> (LCP) 是指標性的時間基準衡量標準，此項會回報最大的 [<span lang="en">above-the-fold element</span>](https://web.dev/lcp/#what-elements-are-considered) 被 <span lang="en">render</span> 所需時間。（註：<span lang="en">above-the-fold element</span> 的意思是，一進到網頁時，不滑動所看到的畫面。<span lang="en">above-the-fold</span> 原意來自於一份未打開的報紙所看到的封面。）
```
謝謝！！

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+
+FID 分數的地理分佈結果證實了上一段綜合裝置圖表的結果。最差還有 79% 網站得到 <span lang="en">Good</span> FID，而韓國則再次以驚人的 97% 在此項評比位居第一。有趣的是，在 CLS 及 LCP 的前幾名競爭者（如：捷克、波蘭、烏克蘭及俄羅斯）反倒在這裡跌至倒數的名次。
+
+再次強調，我們可以推測出為何如此，但我們會需要更多的分析來佐證我們的推測是正確的。如果假設 FID 與 JavaScript 執行性能有關係，在手機性能越好則越貴或甚至被當成奢侈品的國家，反而會得到較低的 FID 排名。波蘭身為 [iPhone 價格最貴](https://qz.com/1106603/where-the-iphone-x-is-cheapest-and-most-expensive-in-dollars-pounds-and-yuan/)國家之一就是個好例子；再加上[平均薪資較低](https://en.wikipedia.org/wiki/List_of_European_countries_by_average_wage#Net_average_monthly_salary)，一份薪水可能還不夠買一隻 Apple 旗艦機。相反地，澳洲的[平均薪資](https:// ...
```
聽起來不錯！謝謝！

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+
+使用者體驗不單單只是個光譜，他更是一個基於重重因素所產生的光譜。為了不能忽視低於效能衡量標準或是無法體驗適當效能的人們，我們一定要設法了解並分析各個面向的數據。每個網頁訪問都述說著各自的故事。我們個人的以及國家層級的經濟狀況也敘述著我們能夠負擔的裝置型態以及網路提供者（<span lang="en">internet provider</span>）。我們生活的地方也深深影響著我們的網路延遲時間（澳洲人時常感受到這個痛苦），而且該國家的經濟也反應了當地行動網路覆蓋率。我們會訪問怎麼樣的網頁呢？我們為什麼想訪問那些網頁呢？了解 context 非常的重要，這不僅僅是為了要分析資料，也是為了能夠開發有足夠同理心、在乎是否容易被訪問且提供了快速的使用者體驗的網頁給所有的使用者。
+
+以粗淺的分析來說，我們可以看到新 <span lang="en">Core WebVitals</span> 效能衡量標準有著樂觀的結果。**如果**我們不將 <span lang="en">Largest Contentful Paint</span> 的數據限縮到 <span lang="en">Poor</span> 體驗以及較慢的網路的話，至少在桌機及手機上，有一半的使用者體驗同時是好的。儘管新的衡量標準現在正有計劃在改善效能問題，但缺少對 FCP (<span lang="en">First Contentful Paint</span>) 以及 TTFB (<span lang="en">Time to First Byte</span>) 的改善方案仍然是個必須面對的嚴重問題。在這裡與 <span lang="en">Largest Contentful Paint</sp ...
```
因為原文是將 if 做成斜體表強調的意思，不過由於之前在 https://github.com/rust-tw/book-tw/pull/6 看到 @weihanglo 分享有關 [大眾字型學 (4)：理想的排版強調法](https://blog.justfont.com/2014/01/popular-typography-4/) 的文章中提到

> 中文沒有斜體，但可以利用有字體家族的字體豐富排版。

所以才採用粗體來強調這邊的「如果」

AbbyTsai replied:
感謝分享喔~

CYBAI commented:
@HTTPArchive/traditional-chinese-translators 以上留言的部分已經在 fixup commit 修正了！再麻煩幫忙校正！謝謝 m(_ _)m

(另外我晚點會再補上 @ArvinH @aszx87410 跟 @andy0130tw 到 contributor 的 configuration！:bow:)

CYBAI commented:
@ArvinH @aszx87410 @andy0130tw 已經幫你們的資料加到 2020 configuration 了！我目前是以你們的 GitHub name 跟 id 來加，另外也只有先幫你們放上 GitHub 帳號，如果有想要放 Twitter、LinkedIn 或是個人網站或是有什麼資料想要改的話，歡迎留言或是用 suggestion！

CYBAI commented on src/config/2020.json:
```diff
@@ -1309,6 +1309,39 @@
       "github": "Zuckjet",
       "twitter": "Zuckjet"
+    },
+    "cybai": {
+      "name": "cybai",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars1.githubusercontent.com/u/6782666?s=460",
+      "github": "CYBAI",
+      "twitter": "_cybai"
+    },
+    "aszx87410": {
+      "name": "Huli",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars1.githubusercontent.com/u/2755720?s=460",
+      "github": "aszx87410"
+    },
+    "ArvinH": {
+      "name": "Huang Shuo-Han",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars1.githubusercontent.com/u/286856?s=460",
+      "github": "ArvinH"
+    },
+    "andy0130tw": {
+      "name": "Andy Pan",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars1.githubusercontent.com/u/5269414?s=460",
+      "github": "andy0130tw"
```
Sure! Will update! Actually I checked if I need to add in alphabetical order but it seems some names are not ordered correctly 😂 maybe it's good to add this in the Translator's Guide 👀?

bazzadp replied:
Oh! Which ones are not alphabetical? Did some sneak through? 😁

CYBAI replied:
I copied the `contributors` and then sort all the keys with `.sort((a, b) => a.localeCompare(b))` and fixed in 20a42ff.

Please help me check if this is what you expected! Thanks!

bazzadp replied:
No that doesn't look right. It's alphabetical by first name so Abbey should be before Adrian but you've swapped them around. Could you revert this back to the way it is in production and then add your names back in again?

Doesn't really need to be in order btw - just annoys me when it isn't! 😁

CYBAI replied:
Ah! I found I misread the first comment here! It's fixed in 358c419 and should be in **first name** alphabetic order now 👀 🙏 Let me know if I'm still wrong 😂

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+authors: [thefoxis]
+reviewers: [borisschapira, rviscomi, obto, noamr, Zizzamia, exterkamp]
+analysts: [max-ostapenko, dooman87]
+translators: [cybai, aszx87410, ArvinH, andy0130tw, AbbyTsai]
```
I see. Will update.

CYBAI commented on src/content/zh-TW/2020/performance.md:
```diff
@@ -0,0 +1,387 @@
+reviewers: [borisschapira, rviscomi, obto, noamr, Zizzamia, exterkamp]
+analysts: [max-ostapenko, dooman87]
+translators: [cybai, aszx87410, ArvinH, andy0130tw, AbbyTsai]
+thefoxis_bio: Karolina 是 <a href="https://calibreapp.com/">Calibre</a> 的產品設計主管。她主要負責打造最全面的速度監測平台。她同時也是為您蒐集效能相關訊息與資訊的 <a lang="en" href="https://perf.email/">Performance Newsletter</a> 發起人之一。Karolina 也<a href="https://calibreapp.com/blog/category/web-platform">時常寫有關於效能如何影響使用者體驗的文章</a>。
```
Awesome! Will update all the links which should be highlighted! Thanks!

AbbyTsai replied:
Nice spotlight. a thought come into my mind as the word might appear many time in chapter. Would we think it is good to highlight all of them in language? or leave them except highlighting the first word. @HTTPArchive/traditional-chinese-translators @bazzadp  would love to hear your idea. thanks.

AbbyTsai replied:
As curiosity, what's a deep reason to declare another language attribute in Chinese, for example: ` <span lang="en">this is English</sapn>`, while it may be shown as string as well. is it? feel free to share your thoughts. thanks.

bazzadp replied:
> a thought come into my mind as the word might appear many time in chapter. Would we think it is good to highlight all of them in language? or leave them except highlighting the first word.

We only highlight links in this way, not English words. It wouldn’t be typical to have multiple links to same place and even if they are used, then yes they ideally all should be highlighted as in a different language from the original article so reader knows before they waste time clicking.

> As curiosity, what's a deep reason to declare another language attribute in Chinese, for example: <span lang="en">this is English</sapn>, while it may be shown as string as well. is it? feel free to share your thoughts. thanks.

This is used to indicate to screen-readers to indicate the language should be read differently (see [WCAG 3.1.2](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts). It is also [possible to use `<i lang="en">` instead of `<span lang="en">`](https://stackoverflow.com/questions/30322839/which-element-should-i-use-for-short-group-of-foreign-langague-words) to show this in italics. Less useful for Chinese where the fact it is in English is more obvious, but can be handy for other Latin languages (e.g. French) to highlight the phrase more as a foreign language phrase.

We strive to have the very best in Accessibility on this site so do recommend translators take these extra steps where at all possible.

CYBAI replied:
Fixed in 7c937c4

bazzadp commented:
Is this ready to merge apart from the contributors issue? Don't understand the Chinese so if you could mark the items as resolved once resolved that would be helpful! 😁

CYBAI commented:
@bazzadp I marked all the comments as resolved and sort the contributors again! Please help me check if the sorting thing is correct or not 🙏 if yes, maybe it's ready to go if there's no other comments from translation reviewers! Thank you!

bazzadp commented:
Thanks all - looks great!

CYBAI commented:
Thanks for all reviewers! 🚀

bazzadp commented:
Just did a release so it is live: https://almanac.httparchive.org/zh-TW/2020/performance

CYBAI commented:
@bazzadp Thank you so much! :D I can read it on the site now!
